onadata/settings/common.py

```diff
@@ -169,8 +169,11 @@
     'django.contrib.messages.middleware.MessageMiddleware',
     'onadata.libs.utils.middleware.HTTPResponseNotAllowedMiddleware',
     'onadata.libs.utils.middleware.OperationalErrorMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
 )
 
+X_FRAME_OPTIONS = 'SAMEORIGIN'
```

Comment:
I think that we should make the default "DENY" as recommended in Django 3.0 - see https://docs.djangoproject.com/en/3.0/ref/clickjacking/

Reply:
Fair point